test/test_pytree.py

```diff
@@ -120,19 +168,23 @@ def run_test(odict):
         od["a"] = torch.tensor(3.14)
         run_test(od)
 
-    def test_flatten_unflatten_namedtuple(self):
+    @parametrize("pytree_impl", [py_pytree, cxx_pytree])
```

Comment:
btw, what does the test name look like?

You might want something more like the following (there is some way to express the output test names)
@parametrize("pytree_impl", [Subtest(py_pytree, name="py"), ...])